src/ray/gcs/subscription_notification.h

```diff
@@ -0,0 +1,53 @@
+#ifndef RAY_GCS_NOTIFICATION_DEF_H
+#define RAY_GCS_NOTIFICATION_DEF_H
+
+#include <ray/protobuf/gcs.pb.h>
+#include <vector>
+
+namespace ray {
+
+namespace gcs {
+
+/// \class SubscriptionNotification
+/// SubscriptionNotification class is a template class which represent
+/// notification of subscription request.
+template <typename Data>
+class SubscriptionNotification {
```

Comment:
Is this class specific to object interface? or is it generic to all the interfaces?

Reply:
This class already renamed to `EntryChangeNotification`. And it's represent notification with `ChangeMode`, is generic to all.